packages/web3-eth-contract/src/contract.ts

```diff
@@ -1116,6 +1116,7 @@ export class Contract<Abi extends ContractAbi>
 					}),
 
 				encodeABI: () => encodeMethodABI(methodAbi, abiParams),
+				decodeABI: (data?: string) => decodeMethodReturn(methodAbi, data),
```

Comment:
Thanks for you contribution,
Could you please, mention this added functionality in the related CHANGELOG.md file?
And could you please add a Unit test for this?

Reply:
@jerkly Thanks